playground/backend/containers/java/Dockerfile

```diff
@@ -15,5 +15,41 @@
 #  See the License for the specific language governing permissions and
 # limitations under the License.
 ###############################################################################
+ARG BASE_IMAGE
+FROM golang:1.17-buster AS build
 
-#Dokerfile to set up the Beam Java SDK
\ No newline at end of file
+# Setup Go Environment
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:$PATH
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1 &&\
+    go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1
+
+# Prepare Application
+COPY src /go/src/playground/backend
+#COPY playground /go/src/playground/playground
+WORKDIR /go/src/playground/backend
+RUN ls
+# Build Application
+RUN go mod download &&\
+    go mod tidy &&\
+    cd cmd/server &&\
+    go build -o /go/bin/server_java_backend
+
+FROM $BASE_IMAGE
+ARG BEAM_VERSION=2.33.0
```

Comment:
somehow i am still seeing the comment and the version hardcoded.

Reply:
That's not hardcoded version, it's default value, that will be overwritten when it will be passed as build arg

Reply:
ah yes, that's true.